CedarJava/src/test/java/com/cedarpolicy/SharedIntegrationTests.java

```diff
@@ -420,9 +420,11 @@ private void executeJsonRequestTest(
             AuthorizationResponse response = auth.isAuthorized(authRequest, slice);
             System.out.println(response.getErrors());
             assertEquals(request.decision, response.getDecision());
-            // convert to a HashSet to allow reordering of error messages
-            assertEquals(new HashSet<>(request.errors), new HashSet<>(response.getErrors()));
+            // convert to a HashSet to allow reordering
             assertEquals(new HashSet<>(request.reasons), response.getReasons());
+            // The integration tests only record the id of the erroring policy, 
+            // not the full error message. So only check that the list lengths match.
```

Comment:
Do we not have access to the ids of the actual erroring policies?

Reply:
The authorization FFI function returns a list of string error messages. Currently, all messages should have the form ``while evaluating `policyX`, ...``, but it feels a little brittle to use a regex to pull out the id. I think checking that the list of errors has the expected length is sufficient for now.